disnake/colour.py

```diff
@@ -319,5 +319,18 @@ def dark_embed(cls) -> Self:
         """
         return cls(0x2B2D31)
 
+    @classmethod
+    def holographic_style(cls) -> Tuple[Self, Self, Self]:
+        """A factory method that returns a tuple of :class:`Colour` with values of
+        ``0xA9C9FF``, ``0xFFBBEC``, ``0xFFC3A0``. This matches the holographic colour style
+        for roles.
+
+        The first value represents the ``colour`` (``primary_color``), the second and the third
+        represents the ``secondary_colour`` and ``tertiary_colour`` respectively.
+
+        .. versionadded:: 2.11
+        """
+        return cls(0xA9C9FF), cls(0xFFBBEC), cls(0xFFC3A0)
+
```

Comment:
Honestly this seems kinda misplaced, "holographic style" is something unique to roles, not any thing that can take in color

Reply:
Maybe moving this as a util method inside `utils.py` would be better, opinions?

Reply:
Since `tertiary_color` is used exclusively for this holographic style, and disallows any other color, maybe just add it to the docstring of `tertiary_color`? That would also make it immediately clear that it can't be used freely for any arbitrary color, unlike primary/secondary